packages/kit/README.md

```diff
@@ -5,10 +5,8 @@ This is the [SvelteKit](https://kit.svelte.dev) framework and CLI.
 The quickest way to get started is via the [create-svelte](https://github.com/sveltejs/kit/tree/main/packages/create-svelte) package:
 
 ```bash
-npm create svelte@latest my-app
-cd my-app
-npm install
-npm run dev
+npm create svelte@latest
```

Comment:
Hmm. I don't know about this change. I think Rich wrote this originally, but this is the set of steps I always follow. The fact that we both do the same thing makes me hesitant to change it. Including the directory name to use makes setup faster and easier and it's a helpful reference for beginners to know what the next commands are. It's possible the creation wizard also prints out that you should run those as subsequent commands, but I think it doesn't hurt to ha e an extra reference for it in either case

Reply:
I've copied the script and the following text from the `create-svelte` package README.

https://github.com/sveltejs/kit/tree/main/packages/create-svelte#readme

Seems like the full script reference is used everywhere else.

Note: The SvelteKit [landing page] seems to be the only one with the `--open` flag included.

[landing page]: https://kit.svelte.dev/

Reply:
We've probably put more thought into the homepage on kit.svelte.dev and svelte.dev than the `create-svelte` readme, so would prefer to align things with the homepage. I personally wouldn't have chosen to have the `--open` flag there, but don't feel strongly about it. Perhaps @Rich-Harris has an opinion about this and the question above (https://github.com/sveltejs/kit/pull/11846#discussion_r1490187624)